src/core/text.ts

```diff
@@ -1,25 +1,11 @@
+import { getGaloyInstanceName, getGaloySMSProvider, yamlConfig } from "@config/app"
```

Comment:
isn't requestPhoneCode part of the login? shouldn't be part of this PR?

Reply:
I planned to do this for the next PR